.github/workflows/build-test.yml

```diff
@@ -156,11 +160,18 @@ jobs:
       id: dyff
       shell: bash
       run: |
-        templates_files=$(find "$RUNNER_TEMP/old" "$RUNNER_TEMP/new" -maxdepth 1 -name '*values.yaml' | sed -E 's|'"$RUNNER_TEMP"'/(old\|new)/||' | sort | uniq)
+        echo "output_dir=$RUNNER_TEMP" | tee -a "$GITHUB_OUTPUT"
+        templates_files=$(find "$RUNNER_TEMP/old" "$RUNNER_TEMP/new" -maxdepth 2 -name '*.yaml' | sed -E 's|'"$RUNNER_TEMP"'/(old\|new)/||' | sort | uniq)
 
         header="# dyff of changes in rendered templates of CI manifests\n\n"
         comment_body=""
         while read -r templates_file; do
+          if [[ "$templates_file" == ".yaml" ]]; then
```

Comment:
How does this happen?

Reply:
`nothing-enabled` values file generated an empty `.yaml` file.